src/components/slice9.js

```diff
@@ -257,5 +258,9 @@ AFRAME.registerComponent("slice9", {
     }
 
     this.setMap(null);
+  },
+
+  remove() {
+    this.geometry?.dispose();
```

Comment:
We were previously leaking this geometry. I noticed it while making sure to clean up the geometry in the new system.